extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java

```diff
@@ -139,5 +313,236 @@ public void registerSwaggerUiServletExtension(SwaggerUiRecorder recorder,
          */
         @ConfigItem(defaultValue = "true")
         boolean enable;
+
+        /**
+         * The urls that will be included as options. By default the OpenAPI path will be used.
+         * Here you can override that and supply multiple urls that will appear in the TopBar plugin.
+         * Format is name=url,name=url (if name is omitted, url will be use as name)
+         */
+        @ConfigItem
+        Optional<String[]> urls;
```

Comment:
I was wondering if maybe we should make it a Map? Or maybe it's more often used without a name?

Reply:
Can you add a Map in MP Config ? Is so that would be better.

Reply:
Yes. See https://github.com/quarkusio/quarkus/blob/master/extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSourceJdbcRuntimeConfig.java#L112

Reply:
Ok cool, how would that look in application.properties ?

Reply:
Like this: 

```properties
quarkus.datasource.jdbc.additional-jdbc-properties.extraProperty1=extraProperty1Value
quarkus.datasource.jdbc.additional-jdbc-properties.extraProperty2=extraProperty2Value 

```

Reply:
Ok cool . That is better. Thanks :)